state/core_access.go

```diff
@@ -99,13 +95,13 @@ func NewCoreAccessor(
 	prt.RegisterOpDecoder(storetypes.ProofOpSimpleMerkleCommitment, storetypes.CommitmentOpDecoder)
 
 	ca := &CoreAccessor{
-		keyring:              keyring,
-		defaultSignerAccount: keyname,
-		getter:               getter,
-		coreIP:               coreIP,
-		grpcPort:             grpcPort,
-		prt:                  prt,
+		keyring:  keyring,
+		getter:   getter,
+		coreIP:   coreIP,
+		grpcPort: grpcPort,
+		prt:      prt,
 	}
+	_ = user.WithDefaultAccount(keyname)
```

Comment:
The default account (& default address) needs to be updated inside `NewCoreAccessor `.

Reply:
Excuse me but why do we need this line?